psc-package.cabal

```diff
@@ -27,7 +27,8 @@ executable psc-package
                    system-filepath -any,
                    text -any,
                    errors -any,
-                   turtle <1.6
+                   turtle <1.6,
+                   SafeSemaphore -any
```

Comment:
No need to depend on this package, the bugs in base's semaphore have been fixed.